.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ""
+labels: bug
+---
+
+---
+
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
```

Comment:
This looks to be repeated from the first part of the file

Reply:
This is the same for
- https://github.com/microsoft/kiota-abstractions-python/pull/199
- https://github.com/microsoft/kiota-http-python/pull/269
- https://github.com/microsoft/kiota-serialization-json-python/pull/211